src/utils/default-content-config.json

```diff
@@ -4,85 +4,123 @@
     "recs": {
       "title": "Recommendations",
       "id": "recommendations",
-      "items": []
-    },
-    "rhel": {
-      "title": "Insights for Red Hat Enterprise Linux",
-      "id": "rhel",
-      "items": [{
-        "id": "rhel-default-1",
-        "permissions": [{
-          "method": "isNotEntitled",
-          "args": ["rhel"]
-        }],
+      "items": [
+        {
+        "id": "remediations-default-1",
         "icon": "play",
         "state": "success",
-        "title": "Red Hat Insights for Red Hat Enterprise Linux simplifies how IT teams maintain and optimize a stable, secure, and performant operating environment.",
+        "title": "Try Streams for Apache Kafka.",
```

Comment:
I think this should be visible before 26th. Can we comment it out for now?

Reply:
I had to remove it, because it is JSON not javascript.

Reply:
Right ofc.